docs/docker.rst

```diff
@@ -22,65 +18,128 @@ Usage
 ^^^^^
 
 The commands below will download the lastest cuda container and run milabench right away,
-storing the results inside the `results` folder on the host machine.
-
-The last command will generate a json summary of each benchmarks.
+storing the results inside the ``results`` folder on the host machine:
 
 .. code-block:: bash
 
+   # Choose the image you want to use
+   export MILABENCH_IMAGE=ghcr.io/mila-iqia/milabench:cuda-nightly
+
    # Pull the image we are going to run
-   sudo docker pull ghcr.io/mila-iqia/milabench:cuda-nightly
+   docker pull $MILABENCH_IMAGE
 
    # Run milabench
-   sudo docker run -it --rm --shm-size=1G                \
-         --runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all  \
-         -v $(pwd)/results:/milabench/envs/runs          \
-         ghcr.io/mila-iqia/milabench:cuda-nightly        \
+   docker run -it --rm --shm-size=8G --gpus=all   \
+         -v $(pwd)/results:/milabench/envs/runs   \
+         $MILABENCH_IMAGE                         \
          milabench run
 
+You may have to increase the value of ``--shm-size`` if it turns out to be insufficient.
```

Comment:
As noted below, this can be simplified using `--ipc=host`, which removes the docker limitations and just uses the normal kernel limits (and also removes isolation).